iOSClient/Supporting Files/en.lproj/InfoPlist.strings

```diff
@@ -1,7 +1,7 @@
 NSCameraUsageDescription = "Camera access is required to scan documents and make photo and video.";
 NSFaceIDUsageDescription = "Face ID is required to authenticate using face recognition.";
-NSLocationAlwaysUsageDescription = "GPS is used to detect new photos from camera roll, continued use of GPS running in the background can dramatically decrease battery life.";
+NSLocationAlwaysUsageDescription = "Location Services are used to detect new photos from camera roll, continued use of Location Services running in the background can dramatically decrease battery life.";
 NSPhotoLibraryUsageDescription = "Photo library access is required to upload your photos and videos to your cloud.";
 NSPhotoLibraryAddUsageDescription = "Photo library access is required to upload your photos and videos to your cloud.";
 NSMicrophoneUsageDescription = "Microphone access is required to create voice notes.";
-NSLocationWhenInUseUsageDescription = "GPS is used to detect new photos from camera roll on background, the use of GPS only when the App is in use is useless.";
+NSLocationWhenInUseUsageDescription = "Location Services are used to detect new photos from the camera roll in the background, the use of Location Services only when the App is in use is useless.";
```

Comment:
```suggestion
NSLocationWhenInUseUsageDescription = "Location Services are used to detect new photos from the camera roll in the background. Using Location Services only when the App is active has no effect.";
```
I'm not sure about `the use [...] in use`. On the other hand *Only when the app is **in use*** is a system message, so it would make sense to use that phrase.